### PR TITLE
Place searchby

### PR DESCRIPTION
**Pull request by O-tega**

### CODE REVIEW

File: app/Controllers/Http/MatchesController.ts [hunk 1]
```diff
@@ -50,10 +50,10 @@ export default class MatchesController {
 		}
 	}
 	public async getMatches({ auth, request, response }: HttpContextContract) {
-		const { accepted } = await request.validate(GetMatchValidator)
 
 		try {
 			const user: User | null = auth.user ?? null
+			const { accepted } = await request.validate(GetMatchValidator)
```
Comment (Okeibunor) [inline app/Controllers/Http/MatchesController.ts:56]: please checkout to dev, pull and merge changes from dev branch. 
`git checkout dev`
`git pull`
`git checkout place-searchby`
`git merge dev`

File: app/Controllers/Http/MatchesController.ts [hunk 2]
```diff
@@ -72,6 +72,8 @@ export default class MatchesController {
 				.preload('breeder')
 				.preload('pet')
 
+				Logger.info('queries', matches)
```
Comment (Okeibunor) [inline app/Controllers/Http/MatchesController.ts:75]: please take this out. Or are you using it?

File: app/Controllers/Http/VetsController.ts [hunk 3]
```diff
@@ -1,3 +1,39 @@
...
 
-export default class VetsController {}
+export default class VetsController {
+	public async vetClinics({ response }: HttpContextContract) {
+		try {
+            const navigate = geolocation.getCurrentPosition
+            const lat = navigate.latitude
+            const lng = navigate.longitude
+
+			const axios = require('axios')
```
Comment (Okeibunor) [inline app/Controllers/Http/VetsController.ts:13]: require statements should be at the top of the file.

File: app/Controllers/Http/VetsController.ts [hunk 4]
```diff
@@ -1,3 +1,39 @@
...
+
+			let config = {
```
Comment (Okeibunor) [inline app/Controllers/Http/VetsController.ts:15]: Create an Axios instance in a separate file. Use [this guide](https://dev.to/ndrohith/creating-an-axios-instance-5b4n) and import it here. The axios instance should have the base url

File: app/Controllers/Http/VetsController.ts [hunk 5]
```diff
@@ -1,3 +1,39 @@
...
+				method: 'get',
+				url: 'https://maps.googleapis.com/maps/api/place/nearbysearch/json?location=-33.8670522%2C151.1957362&radius=1500&type=veterinary_care&keyword=vet_clinics&key=YOUR_API_KEY',
```
Comment (Okeibunor) [inline app/Controllers/Http/VetsController.ts:17]: Why are there latitude, longitude and radius numbers here? Where are they from?

Comment (O-tega) [reply app/Controllers/Http/VetsController.ts:17]: I did

File: app/Services/apis/googleMapApi.ts
[the remark this answers is not in the document]
Comment (O-tega) [reply app/Services/apis/googleMapApi.ts:1]: I did

File: app/Controllers/Http/VetsController.ts [hunk 6]
```diff
@@ -1,32 +1,27 @@
...
-
-			let config = {
-				method: 'get',
-				url: 'https://maps.googleapis.com/maps/api/place/nearbysearch/json?location=-33.8670522%2C151.1957362&radius=1500&type=veterinary_care&keyword=vet_clinics&key=YOUR_API_KEY',
-				headers: {},
-                // The google url has a default longitude and latitude so I did this, I don't know if its correct though I couldn't find such in the documentation
-				url2: `https://maps.googleapis.com/maps/api/place/nearbysearch/json?location=${lat}%${lng}&radius=1500&type=veterinary_care&keyword=vet_clinics&key=YOUR_API_KEY`,
-			}
+			const getNearbyLocation = await nearbySearch({
+				url: `location=${lat}%${lng}&radius=1500&type=veterinary_care&keyword=vet_clinics&key=YOUR_API_KEY`,
```
Comment (Okeibunor) [inline app/Controllers/Http/VetsController.ts:15]: the `YOUR_API_KEY` should be an environment variable in the .env file.

**Pull request closed without merging**